src/identity/interaction/util/BasicIssuerReferenceWebIdOwnershipValidator.ts

```diff
@@ -0,0 +1,68 @@
+import { literal, namedNode, quad } from '@rdfjs/dataset';
+import fetch from '@rdfjs/fetch';
+import type { DatasetResponse } from '@rdfjs/fetch-lite';
+import type { DatasetCore } from 'rdf-js';
+import { getLoggerFor } from '../../../logging/LogUtil';
+import { SOLID } from '../../../util/Vocabularies';
+import { WebIdOwnershipValidator } from './WebIdOwnershipValidator';
+
+/**
+ * Validates is a WebId is okay to register based on if it
+ * references this as an issuer.
+ *
+ * Note: This file uses clownface because I like clownface.
+ * If needed, we can refactor it to use whatever rdf library
+ * Ruben prefers.
```

Comment:
@rubensworks Thoughts on `@rdfjs/fetch` versus what we use elsewhere?

Reply:
I've removed it, just not the comment. I removed the comment now.